src/parser/semantic_check.c

```diff
@@ -8194,7 +8194,7 @@ pt_check_create_entity (PARSER_CONTEXT * parser, PT_NODE * node)
   DB_OBJECT *db_obj, *existing_entity;
   int found, partition_status = DB_NOT_PARTITIONED_CLASS;
   int collation_id, charset;
-  bool found_reuse_oid = false;
+  bool found_reuse_oid = false, reuse_oid = true;
```

Comment:
it looks better to rename found_reuse_oid to found_reuse_option for readability.